doc/installation.rst

```diff
@@ -178,52 +172,39 @@ we'll also call it the ``<dipy root>`` directory.
 Building and installing
 =======================
 
-Windows
--------
-
-Anaconda_ is probably the easiest way to install the dependencies that you need.
+.. _install-source-nix:
 
-Start a command shell like ``cmd`` or Powershell_ and change directory into the
-*dipy source root directory*.
+Install from source for Unix (e.g Linux, OSX)
+---------------------------------------------
 
-To install into your system::
+Change directory into the *dipy source root directory* .
 
-    python setup.py install --compiler=mingw32
+To install for the system::
 
-To install inplace - so that dipy is running out of the source code directory::
+    python setup.py install
 
-    python setup.py develop
+To build dipy in the source tree (locally) so you can run the code in the source tree (recommended for following the latest source) run::
 
-(this is the mode we recommend for following the latest source code).  If you
-get an error with ``python setup.py develop`` make sure you have installed
-`setuptools`_.
+    python setup.py build_ext --inplace
 
-If you get an error saying  "unable to find vcvarsall.bat" then you need to
-create a file called "pydistutils.cfg" in notepad and give it the contents ::
+add the *dipy source root directory* into your ``PYTHONPATH`` environment variable. Search google for ``PYTHONPATH`` for details or see `python module path`_ for an introduction.
 
-  [build]
-  compiler=mingw32
+When adding dipy_ to the ``PYTHONPATH``, we usually add the ``PYTHONPATH`` at
+the end of ``~/.bashrc`` or (OSX) ``~/.bash_profile`` so we don't need to
+retype it every time. This should look something like::
 
-Save this into your system python ``distutils`` directory as ``distutils.cfg``.
-This will be something like ``C:\Python26\Lib\distutils\distutils.cfg``.
+  export PYTHONPATH=/home/user_dir/Devel/dipy:/home/user_dir/Devel/nibabel
```

Comment:
It's not necessary to have nibabel in the PYTHONPATH. But you need to have nibabel installed.